python/ray/autoscaler/gcp/example-full.yaml

```diff
@@ -115,6 +115,10 @@ available_node_types:
             # Comment this out to use on-demand.
             scheduling:
               - preemptible: true
+            # Un-Comment this to launch workers with the Service Account of the Head Node
+            # - email: ray-autoscaler-sa-v1@<project_id>.iam.gserviceaccount.com
```

Comment:
Typo: Missing serviceAccounts field key above this (looks like unconnected stuff would land under scheduling)

Reply:
Ah yep! Thanks!!

Reply:
```suggestion
            # serviceAccounts:
            # - email: ray-autoscaler-sa-v1@<project_id>.iam.gserviceaccount.com
```